### PR TITLE
chore: prepend @aws-amplify to dependencies pt.3

### PR DESCRIPTION
**Pull request by evcodes**
#### Description of changes
Dependencies Prepending aws-amplify to :
- cli-shared-interfaces
- amplify-console-hosting
- amplify-container-hosting
- amplify-dotnet template provider.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [n/a] Relevant documentation is changed or added (and PR referenced)
- [n/a] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [x] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by evcodes** [comment]
this passed e2e tests but failed lint tests because this was out of date with dev branch

File: packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* eslint-disable class-methods-use-this */
-import { ICommandInput, IFlowData } from 'amplify-cli-shared-interfaces';
-import { IFlowReport } from 'amplify-cli-shared-interfaces/lib/amplify-cli-flow-reporter-types';
+import { ICommandInput, IFlowData } from '@aws-amplify/amplify-cli-shared-interfaces';
+import { IFlowReport } from '@aws-amplify/amplify-cli-shared-interfaces/lib/amplify-cli-flow-reporter-types';
```
Comment (sobolk) [inline packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts:3]: we should fix this `lib` import.

**Review by goldbez — APPROVED** [review]

File: packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts (hunk 1 above)
Comment (evcodes) [reply packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts:3]: how would we fix? Leaving as it was before?

Comment (sobolk) [reply packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts:3]:
```suggestion
import { IFlowReport } from '@aws-amplify/amplify-cli-shared-interfaces';
```

File: packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /* eslint-disable class-methods-use-this */
-import { ICommandInput, IFlowData } from 'amplify-cli-shared-interfaces';
-import { IFlowReport } from 'amplify-cli-shared-interfaces/lib/amplify-cli-flow-reporter-types';
+import { ICommandInput, IFlowData } from '@aws-amplify/amplify-cli-shared-interfaces';
+import { IFlowReport } from '@aws-amplify/amplify-cli-shared-interfaces';
```
Comment (jhockett) [inline packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts:3]: These should be combined into a single import

**Review by jhockett — APPROVED** [review]

File: packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts (hunk 2 above)
Comment (evcodes) [reply packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts:3]: yes
```suggestion
import { ICommandInput, IFlowData, IFlowReport } from '@aws-amplify/amplify-cli-shared-interfaces';
```

**Comment by evcodes** [comment]
addressed all comments, would appreciate another round of eyes 👍🏼

**Review by sobolk — APPROVED** [review]

File: packages/amplify-cli/package.json [hunk 3]
```diff
@@ -52,13 +52,13 @@
...
+    "@aws-amplify/amplify-cli-shared-interfaces": "1.1.1",
     "@aws-amplify/amplify-cli-logger": "1.2.3",
-    "amplify-cli-shared-interfaces": "1.1.1",
     "amplify-codegen": "^3.3.6",
-    "amplify-console-hosting": "2.4.1",
-    "amplify-container-hosting": "2.5.11",
+    "@aws-amplify/amplify-console-hosting": "2.4.1",
+    "@aws-amplify/amplify-container-hosting": "2.5.10",
+    "@aws-amplify/amplify-dotnet-function-template-provider": "2.4.3",
     "amplify-dotnet-function-runtime-provider": "1.7.1",
```
Comment (jhockett) [inline packages/amplify-cli/package.json:61]: amplify-dotnet-function-runtime-provider is missing the `@aws-amplify/` prefix

**Pull request closed without merging**